benches/bench.rs

```diff
@@ -1560,7 +1657,11 @@ fn bench_analysis_storage(c: &mut Criterion) {
     for input_size in &INPUT_SIZES {
         let contract_identifier = QualifiedContractIdentifier::local(&*format!("c{}", 0)).unwrap();
 
-        let (_, contract) = gen(function, 1, *input_size);
+        let GenOutput {
+            setup: _,
+            body: contract,
+            input_size: _,
+        } = gen(function, 1, *input_size);
```

Comment:
I think we should be passing `SCALE` as the second arg here (based on a look at this function and the corresponding generator)